packages/pinia/src/store.ts

```diff
@@ -866,6 +866,7 @@ export function defineStore(
     options = idOrOptions
     id = idOrOptions.id
   }
+  // push id into stores array
```

Comment:
This comment is confusing here,  should be on the line before the action its describing, or be removed

Reply:
Thanks, i remove useless comment!